<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inv</class>
 <widget class="QWidget" name="inv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="invLabel">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>INVENTORY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="burnButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
    </font>
   </property>
   <property name="text">
    <string>BURN</string>
   </property>
  </widget>
  <widget class="QPushButton" name="eatButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>330</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
    </font>
   </property>
   <property name="text">
    <string>EAT</string>
   </property>
  </widget>
  <widget class="QListWidget" name="itemsList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>441</width>
     <height>521</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="tossButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>151</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
    </font>
   </property>
   <property name="text">
    <string>TOSS OVERBOARD</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Andale Mono</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{ 
				background-color : rgb(168, 168, 168);
				color : rgb(255, 255, 255)
			 }</string>
   </property>
   <property name="text">
    <string>Your Inventory.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
